prime-router/src/main/kotlin/azure/QueueAccess.kt

```diff
@@ -66,13 +66,14 @@ object QueueAccess {
             invisibleDuration
         }
         val timeToLive = invisibleDuration.plusDays(timeToLiveDays)
-        createQueueClient(queueName).sendMessageWithResponse(
+        val response = createQueueClient(queueName).sendMessageWithResponse(
             message,
             duration,
             timeToLive,
             null,
             null
         )
+        return response.value.messageId
```

Comment:
Nice! Given an Azure Error Event, this allows us to easily find the associated queue message in the poison queue.